src/neuroconv/nwbconverter.py

```diff
@@ -184,3 +183,42 @@ def run_conversion(
                 )
 
         return nwbfile_out
+
+
+class NWBConverterPipe(NWBConverter):
+    """Takes pre-initialized interfaces as arguments to build an NWBConverter class"""
+
+    def get_conversion_options_schema(self):
```

Comment:
Should maybe also do

```python
def get_source_schema():
     raise NotImplementedError("Informative error about why this doesn't make sense here...")
```

to avoid having a confusing method available for the class?

Reply:
You can get the source schema if you want.

Reply:
But how? That's a class method as well and won't mean anything since the `__init__` here doesn't take `source_data`, so what is the meaning?

Reply:
Oh, what I mean is that we could have the equivalent method that returns the source schema as that is extracted by the data classes which are available here as well.  The problem will be `_validate_source_data` that requires `source_data`.

Reply:
Yeah I mean you *could* have it defined as an instance method that tells you what the input keyword argument *were* for the already passed interfaces, it just seems redundant at that point though since the person obviously knows how to instantiate the interfaces to begin with

Reply:
I'd just recommend removing it through, w.r.t. this new subclass any method having to do with source data/schema just doesn't make sense

Reply:
The only thing that I was saying is that `get_source_schema` makes sense here (even it is not useful). The one that does not make sense is `_validate_source_data` as that requires `source_data`. 

I agree it would be better to remove it. Any suggestions for how to go about? To me this indicates that there should be a third class from were both `NWBConveter` and the class proposed here both inherit from. Maybe there is a simpler way.